packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -286,23 +303,49 @@ abstract class FlameChartState<T extends FlameChart, V> extends State<T>
         zoomTo(math.max(FlameChart.minZoomLevel,
             zoomController.value - keyboardZoomOutUnit));
       } else if (keyLabel == 'a') {
-        scrollTo(
+        horizontallyScrollTo(
             linkedHorizontalScrollControllerGroup.offset - keyboardScrollUnit);
       } else if (keyLabel == 'd') {
-        scrollTo(
+        horizontallyScrollTo(
```

Comment:
horizontallyScrollTo is an ugly name.
how about just scrollToX and scrollToY
or
scrollBy(deltaX, deltaY) or
scrollBy(Offset)